### DISCUSSION TITLE
Safe Video

### DISCUSSION THREAD

**Discussion by marcosffm** [aclap-dev/video-downloadhelper#535] [Help and Q&A]
### Your question

Como realizar download com a proteção "safe video"

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://app.nutror.com/curso/2dc3d25c2a3f87da5242f84ca7d7334c9f89c0e4/aula/8936817",
  "error": {
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\r\n    Last message repeated 7 times\r\n[in#0 @ 00000190ff5eb980] Error opening input: Invalid data found when processing input\r\nError opening input file https://subplaylist-videozz.k8s.eduzz.com/?t=eyJhbGciOiJIUzI1NiJ9.eyJrZXkiOiI1YzIzYjQwMS01MjI3LTQzZTktYmQ2My0xMDkxNmYzYWNlZTEiLCJzZXNzaW9uS2V5Ijoib2hpN3Z2d2UiLCJwcm9kdWNlcktleSI6IjM4MTY0ODI1IiwicHJvZHVjZXJFbWFpbCI6Imd1dG9jbDE5NjlAZ21haWwuY29tIiwiY29uc3VtZXJFbWFpbCI6Im1hcmNvc2ZmbUBnbWFpbC5jb20iLCJpbXByaW50Ijp0cnVlLCJzdG9yYWdlIjoicjIiLCJwbCI6IjcyMHBfcGxheWxpc3QubTN1OCIsImlhdCI6MTcxNjM3NTkyNywiZXhwIjoxNzE2NTQ4NzI3fQ.m-330LbXFquIKqqhoYA8korcHS5673gqagPo0_bbmzc.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2404964710526760",
    "error": "coapp_failure",
    "id": "coapp_failure_99150cfe-5f5d-46c1-842a-70eaf6eccb8d"
  }
}

**Comment by GrampaWildWilly** [comment]
Take a look at #521.
